miden-tx/src/tests.rs

```diff
@@ -172,22 +162,25 @@ fn executed_transaction_account_delta() {
             ## Send some assets from the account vault
             ## ------------------------------------------------------------------------------------
             # partially deplete fungible asset balance
-            push.0.1.2.3
-            push.999
-            push.{REMOVED_ASSET_1}
-            call.wallet::send_asset drop dropw dropw
+            push.0.1.2.3            # recipient
+            push.{PUBLIC_NOTE}      # note_type
+            push.999                # tag
+            push.{REMOVED_ASSET_1}  # asset
+            call.wallet::send_asset dropw dropw drop drop
 
             # totally deplete fungible asset balance
-            push.0.1.2.3
-            push.999
-            push.{REMOVED_ASSET_2}
-            call.wallet::send_asset drop dropw dropw
+            push.0.1.2.3            # recipient
+            push.{PUBLIC_NOTE}      # note_type
+            push.998                # tag
+            push.{REMOVED_ASSET_2}  # asset
+            call.wallet::send_asset dropw dropw drop drop
 
             # send non-fungible asset
-            push.0.1.2.3
-            push.999
-            push.{REMOVED_ASSET_3}
-            call.wallet::send_asset drop dropw dropw
+            push.0.1.2.3            # recipient
+            push.{PUBLIC_NOTE}      # note_type
+            push.997                # tag
+            push.{REMOVED_ASSET_3}  # asset
+            call.wallet::send_asset dropw dropw drop drop
```

Comment:
It seems that removing the `dropw dropw drop drop` would be a bigger issue, namely defining the calling convention. For now this makes the tests happy.